BookingsApi/BookingsApi/Controllers/V2/HearingsControllerV2.cs

```diff
@@ -247,7 +247,45 @@ await UpdateHearingDetails(hearing.Id, requestHearing.ScheduledDateTime,
 
             return NoContent();
         }
+        
+        /// <summary>
+        /// Return hearing details for todays hearings
+        /// </summary>
+        /// <returns>Booking status</returns>
+        [HttpGet("today")]
+        [OpenApiOperation("GetHearingsForToday")]
+        [ProducesResponseType(typeof(List<HearingDetailsResponseV2>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [MapToApiVersion("2.0")]
+        public async Task<IActionResult> GetHearingsForToday()
+        {
+            var videoHearings = await _queryHandler.Handle<GetHearingsForTodayQuery, List<VideoHearing>>(new GetHearingsForTodayQuery());
+            if (!videoHearings.Any())
```

Comment:
To reduce our 404s we should treat our empty lists as a 200 with an empty list. This will also reduce the code on the client side to handle empty lists. There's a ticket on the backlog to refactor old endpoints, so we should try to avoid adding new ones that return a 404 for empty lists